tgui/packages/tgui/styles/components/Flex.scss

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2020 Aleksej Komarov
+ * SPDX-License-Identifier: MIT
+ */
+
+.Flex {
+  display: -ms-flexbox;
```

Comment:
We should probably go and look for an appropriate `autoprefixer` that we can add in to our build chain to do this sort of thing for us.

Reply:
No, because it's extra bloat, and we only need to support IE11.

Reply:
That's fair. Would prefer done as a mixin, then.